src/event-helper.js

```diff
@@ -40,6 +40,32 @@ export function listen(element, eventType, listener, opt_capture) {
 
 
 /**
+ * Globally Listens for the specified event for the specified tag name.
+ * @param {!Window} win
+ * @param {string} tagName
+ * @param {string} eventType
+ * @param {function(Event)} listener
+ * @param {boolean=} opt_capture
+ * @return {!UnlistenDef}
+ */
+export function listenForEventOnTagName(
+    win, tagName, eventType, listener, opt_capture) {
+  const capture = opt_capture || false;
+  win.addEventListener(eventType, event => {
+    if (event.target.tagName.toLowerCase() == tagName.toLowerCase()) {
+      listener(event);
+    }
+  }, capture);
+  return () => {
+    if (win) {
+      win.removeEventListener(eventType, listener, capture);
```

Comment:
This won't work since `listener` here is not the same thing you used in `addEventListener`. 

And as asked above, I'm not sure if this helps vs just doing plain click listeners on the elements themselves: this listens to a lot more events then we need. Also, we would generally like not to make it too easy to add global event listeners, e.g. for "touchstart" and many others.